test-requirements.txt

```diff
@@ -14,19 +14,19 @@ codecov~=2.0.5
 freezegun~=0.3.9
 moban~=0.5.0 ; python_version > '3.0'
 pytest~=3.6.4
-pytest-cov~=2.4
+pytest-cov>=2.4.0,<2.6
 pytest-env~=0.6.0
 pytest-error-for-skips~=1.0
 pytest-instafail~=0.3.0
 pytest-mock~=1.1
 pytest-profiling~=1.3.0
 pytest-reorder~=0.1.0
-git+https://github.com/jayvdb/pytest-reqs@coala#egg=pytest-reqs
+pytest-reqs==0.1
 pytest-timeout~=1.3.0
 pytest-travis-fold~=1.3.0
 requests-mock~=1.2
 ipdb~=0.11
-pip<10
+pip
```

Comment:
No, this won't work. There is a hard check on pip version, coala won't work with anything above pip 9. There is a hard check on coala setup.py too to check this.

Reply:
Umm, but it is working fine for me. As far as I understand, there is no such hard check for pip, but pip 9 was being used so that setup-tools can be downgraded; there is a hard check on that in coala/.misc/check_setuptools.py

Reply:
Yes, there is. See https://github.com/coala/coala/blob/master/setup.py#L441

Reply:
Is there any specific reason for that because I can see that even in logs of the previous run [link](https://travis-ci.org/github/coala/coala/jobs/682807502) that pip-18.1(line252) was used.

Reply:
Yes, a lot of reasons. I couldn't possibly list them down here. This is what we will be changing in the python update project.

Reply:
Okay, while exploring bears repo I understood your concerns, will make the necessary changes.